cli/azd/pkg/tools/powershell/powershell.go

```diff
@@ -25,9 +26,21 @@ type powershellScript struct {
 	envVars       []string
 }
 
+func checkInstalled(options tools.ExecOptions) (err error) {
+	return tools.ToolInPath(options.UserPwsh)
+}
+
 // Executes the specified powershell script
 // When interactive is true will attach to stdin, stdout & stderr
 func (bs *powershellScript) Execute(ctx context.Context, path string, options tools.ExecOptions) (exec.RunResult, error) {
+	if err := checkInstalled(options); err != nil {
+		return exec.RunResult{}, &internal.ErrorWithSuggestion{
+			Err: err,
+			Suggestion: "PowerShell is not installed or not in the path. To install PowerShell, " +
+				"visit https://learn.microsoft.com/powershell/scripting/install/installing-powershell",
```

Comment:
Can we use `WithLinkFormat` to format the link?

https://github.com/Azure/azure-dev/blob/e63055e132c80705ba531c8190daf4188619c1d2/cli/azd/pkg/output/colors.go#L12-L15